packages/cli/src/ui/components/SettingsDialog.tsx

```diff
@@ -768,9 +867,17 @@ export function SettingsDialog({
       height="100%"
     >
       <Box flexDirection="column" flexGrow={1}>
-        <Text bold={focusSection === 'settings'} wrap="truncate">
-          {focusSection === 'settings' ? '> ' : '  '}Settings
-        </Text>
+        {isSearching || searchQuery ? (
+          <Text bold color={theme.text.accent} wrap="truncate">
+            {isSearching ? '> ' : '  '}Search: {searchQuery}
+            {isSearching ? '_' : ''}
```

Comment:
we should align with other input prompt UI which uses an inverted character instead of _`
see ApiAuthDialog which uses TextBuffer to for a single line text input. Ideally you would refactor this code and ApiAuthDialog to use a standard single line text input.